admin-server/go/pkg/file/upload_processor.go

```diff
@@ -187,6 +188,7 @@ func processAndStoreUpload(upload *model.Upload, file *os.File) (uploadProcessRe
 	batchSize := 0                      // cumulative batch size in bytes
 	maxMutationSize := 16 * 1024 * 1024 // 16MB
 	safetyMargin := 0.75
+	maxCellSize := 1200 //1MB
```

Comment:
I missed this before but this isn't 1MB. 1MB = 1024 * 1024 bytes:
```suggestion
	maxCellSize := 1024 * 1024 // 1MB
```